clients/drcachesim/simulator/tlb_simulator.cpp

```diff
@@ -94,46 +96,47 @@ tlb_simulator_t::tlb_simulator_t(const tlb_simulator_knobs_t &knobs)
         lltlbs_[i] = NULL;
     }
     for (unsigned int i = 0; i < knobs_.num_cores; i++) {
-        itlbs_[i] = create_tlb(knobs_.TLB_replace_policy);
-        if (itlbs_[i] == NULL) {
-            error_string_ = "Failed to create itlbs_";
-            success_ = false;
-            return;
-        }
-        dtlbs_[i] = create_tlb(knobs_.TLB_replace_policy);
-        if (dtlbs_[i] == NULL) {
-            error_string_ = "Failed to create dtlbs_";
-            success_ = false;
-            return;
-        }
-        lltlbs_[i] = create_tlb(knobs_.TLB_replace_policy);
-        if (lltlbs_[i] == NULL) {
-            error_string_ = "Failed to create lltlbs_";
-            success_ = false;
-            return;
-        }
-
+        std::stringstream ss_itlb;
+        ss_itlb << "itlb " << i;
+        itlbs_[i] = new tlb_t(ss_itlb.str());
```

Comment:
Alternately, I _think_ you could have done something like:
```suggestion
        std::string i_str = std::to_string(i);
        itlbs_[i] = new tlb_t("itlb" + i_str);
        dtlbs_[i] = new tlb_t("dtlb" + i_str);
        ...
```

Reply:
done.